python/seldon_core/proto/prediction_pb2.py

```diff
@@ -731,99 +731,99 @@
 DESCRIPTOR.message_types_by_name['RequestResponse'] = _REQUESTRESPONSE
 _sym_db.RegisterFileDescriptor(DESCRIPTOR)
 
-SeldonMessage = _reflection.GeneratedProtocolMessageType('SeldonMessage', (_message.Message,), dict(
-  DESCRIPTOR = _SELDONMESSAGE,
-  __module__ = 'proto.prediction_pb2'
+SeldonMessage = _reflection.GeneratedProtocolMessageType('SeldonMessage', (_message.Message,), {
+  'DESCRIPTOR' : _SELDONMESSAGE,
```

Comment:
Is this change because we're using an update build of proto or how come?

Reply:
yeah, this doesn't seem good if we need to regenerate the proto -> python

Reply:
@cliveseldon Would it be better to use the build of `protoc` used originally to generate these in case there are breaking changes somewhere?